scripts/nullfs.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+## Script to install and configure nullfs to force-deactivate public param caching
+
+# Install nullfs
+git clone https://github.com/abbbi/nullfsvfs.git
+cd nullfsvfs
+sudo apt-get install debhelper dkms
+sudo dpkg-buildpackage -r
+# Update below version as needed
+sudo dpkg -i ../nullfsvfs_0.17_amd64.deb
+
+# Set up nullfs
+sudo modprobe nullfs
+echo nullfs | sudo tee /etc/modules-load.d/nullfs.conf
+NFS_USERNAME=$(whoami)
```

Comment:
Nit: NFS usually stands for [something else](https://en.wikipedia.org/wiki/Network_File_System) in a Unix context, this acronym may surprise.